elifetools/tests/test_parse_jats.py

```diff
@@ -3746,6 +3746,24 @@ def test_refs(self, filename):
                 "id": "bib15"
             }
         ]
+        ),
+
+         # example of mixed-citation with string-name, based on non-elife article
+        ('<root xmlns:xlink="http://www.w3.org/1999/xlink"><article><back><ref-list><ref><mixed-citation publication-type="book" meta="no" id="c2"><person-group person-group-type="author"><string-name><surname>Frank</surname>, <given-names>A. W.</given-names></string-name></person-group> (<year>1995</year>). <source>The wounded storyteller: Body, illness, and ethics</source>. <publisher-loc>Chicago</publisher-loc>: <publisher-name>University of Chicago Press</publisher-name>.</mixed-citation></ref></ref-list></back></article></root>',
```

Comment:
I can't wait for this XML to go into a file where newlines can be used (out of scope for this PR)

Reply:
Sometimes newlines will add unwanted whitespace though, since the XML we typically read in is all on one line. Though most cases in the future can probably have multi-line input, it will depend on whether we get the right output from it.